### ISSUE TITLE
Please update Fedora logo

### ISSUE THREAD

**Issue by mattdm** [simple-icons/simple-icons#6979]
### Brand Name

Fedora

### Official Resources for Icon and Color

SVG in zip file: [fedora_fa.zip](https://github.com/simple-icons/simple-icons/files/7771633/fedora_fa.zip)

The "new" Fedora blue is `#51a2da`.

Official guidelines will be at https://docs.fedoraproject.org/en-US/project/brand/, but aren't ready yet; I didn't want to wait on that in getting this to you.

### Additional Comments

I have clearance from the Fedora Council (as stewards of the trademark) and our legal for it to be in Simple Icons officially.

----

The Fedora Council grants permission for a version of the official Fedora logo to be included in your collection of icons, unmodified from the logo image we are providing here except in size and/or file format, provided you include this text, the legal notice below, and a link to https://docs.fedoraproject.org/en-US/project/brand/ in any relevant accompanying documentation and websites. Please do not create modified versions of the logo, even to match a style or theme.

ACKNOWLEDGMENT FOR LOGO USE:

The "Fedora" logo provided as part of this collection is a trademark of Red Hat, Inc. and is provided solely for use to accurately identify Fedora or its products. To use this logo, you must agree to only use the logo in compliance with the Fedora Trademark Guidelines: https://fedoraproject.org/wiki/Legal:Trademark_guidelines, and never to use any logo or trademark of Red Hat in a manner that suggests that Red Hat or Fedora owns, sponsors, or endorses your products or services, or that you own any rights in the logo or trademark.

----

I hope this works for you. Let me know what I can do to help.

### Contributing

Yes

**Comment by sachinraja** [comment]
The text indicates that you will not be making a PR, but you marked "Yes" in the contributing section, can you clarify?
> unmodified from the logo image we are providing here except in size and/or file format

Does this mean we cannot apply any optimizations?

> provided you include this text, the legal notice below

We need a link to include that legal notice, preferably to the license.

**Comment by mattdm** [comment]
> The text indicates that you will not be making a PR, but you marked "Yes" in the contributing section, can you clarify?

I misunderstood the meaning of the question — it said "This is an open source project and we welcome contributions. Do you want to update this icon?". I _do_ want the icon to be updated, so I said "yes".

I can create PR.


> Does this mean we cannot apply any optimizations?

Consider optimizations to the file which don't cause it to be rendered differently as just a change to the file format. That's fine. Optimizations which would cause it to be drawn differently are not. (E.g., replacing the bubble shape with a circle.)

> We need a link to include that legal notice, preferably to the license.

The statement will be included in https://docs.fedoraproject.org/en-US/project/brand/. (If you need it there before we can move forward, I can update that page.) The logo is not open source and there is not a license other than the above text.

**Comment by sachinraja** [comment]
We don't have a field for just the acknowledgement, but we can just say that the license type is custom and link to that. Everything here looks good so creating a PR would be great :+1:.

**Comment by service-paradis** [comment]
It would be good to review #5540 to be sure before going forward with a new PR.

**Comment by sachinraja** [comment]
Looks like we have legal approval now, so we can go ahead with it. Reviewed the icon in #5610 and it matches perfectly so I'll reopen that.

**Comment by mattdm** [comment]
Ah, cool. Yes, that looks right at first glance. Thank you.
